priv/mssql2012.sql

```diff
@@ -212,9 +212,9 @@ CREATE TABLE [dbo].[privacy_list](
 	[name] [nvarchar](250) NOT NULL,
 	[id] [bigint] IDENTITY(1,1) NOT NULL,
 	[created_at] [datetime] NOT NULL,
- CONSTRAINT [privacy_list$id] UNIQUE CLUSTERED
+ CONSTRAINT [PK_privacy_list] PRIMARY KEY CLUSTERED
 (
-	[id] ASC
+	[server], [username]
```

Comment:
I am not sure why we kept `id` as the primary key here... but I think `name` should be included in the index/primary key at least, just like for the other tables.